index.html

```diff
@@ -7,13 +7,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Victoria's Portfolio</title>
     <link rel="stylesheet" href="style.css"/>
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
-    />
   </head>
 
   <body>
+  <!-- html for the pop ups -->
+
+  <div class="modal">
+
+  <div class="pop-up-wrap">
+    <div class="pop-up-nav">
+      <h2 class="poppins" id="logo">Tonic</h2>
+      <span class = "pop-up-closeBtn">&times;</span>
+    </div>
+
+  <div class="pop-up-about">
+      <ul class="poppins canopy-modal">
+        <li class="about-li" id="canopy">CANOPY</li>
+        <li class="about-li">Back End Dev</li>
+        <li class="about-li">2015</li>
+      </ul>
+  </div>
+
+  <div class="pop-up-image-wrap">
+    <img class="snapshoot" src="./assets/project1.png" alt="Tanner Christensen's portfolio image">
+  </div>
+     
+  <div class="pop-up-text-content">
+    <p class="poppins-regular" id="modal-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div class="tag-wrap">
+    <ul class="poppins-regular language-button">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+  <div class="pop-up-btn">
+    <button class="poppins-regular see-live-btn">See live<img class="live-icon" src="./assets/live-icon.png"/></button>
+    <button class="poppins-regular see-source-btn">See source<img id="github-modal-icon" src="./assets/github-icon.png"/></button>
+  </div>
+  </div>
+  </div>
+</div>
+</div>
+
+<!-- End of html for pop up -->
```

Comment:
- Hey great job with the pop however, according to the project requirement, you're to create the pop-up dynamically using javascript. please kindly use adjust your implementation to meet the project requirements

Reply:
- Also, you're to update the project section in the index.htm file by adding them dynamically using javascript. i.e, the HTML structures you have hard coded in the index.html file should no longer be there but created dynamically using javascript. Therefore, these [lines of codes](https://github.com/Victoria129/Portfolio/blob/4958428085c86e937af2aa167f0c40a00a28c42a/index.html#L99-L190) should be added dynamically. please kindly update this to meet the project requirement